config/api.yml

```diff
@@ -2315,6 +2316,53 @@
       :post:
       - :name: remove
         :identifier: svc_catalog_provision
+  :cloud_templates:
+    :description: Cloud Templates
+    :identifier: template
+    :options:
+      - :collection
+      - :subcollection
+    :verbs: *gpd
+    :klass: ManageIQ::Providers::CloudManager::Template
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show_list
+      :post:
+      - :name: query
+        :identifier: miq_template_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show
+      :post:
+      - :name: delete
+        :identifier: image_delete
+      - :name: query
+        :identifier: miq_template_show
+      - :name: create
+        :identifier: image_create
+    :subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show_list
+      :post:
+      - :name: delete
```

Comment:
Can you add tests for these subcollection actions?

Reply:
@jntullo added tests for provider's image, and removed `delete` action for now